hack/jenkins/e2e-image/Makefile

```diff
@@ -15,6 +15,8 @@
 IMG = gcr.io/google-containers/kubekins-e2e
 TAG = $(shell date +v%Y%m%d)
 
+all: push
```

Comment:
I agree, though this is the pattern that basically every one of our Docker Makefiles uses.

This is one of the reasons our normal accounts don't have push access. :)